pkg/cache/v3/linear_test.go

```diff
@@ -73,16 +62,10 @@ type resourceInfo struct {
 func validateDeltaResponse(t *testing.T, resp DeltaResponse, resources []resourceInfo, deleted []string) {
 	t.Helper()
 
-	if resp.GetDeltaRequest().GetTypeUrl() != testType {
-		t.Errorf("unexpected empty request type URL: %q", resp.GetDeltaRequest().GetTypeUrl())
-	}
+	assert.Equalf(t, testType, resp.GetDeltaRequest().GetTypeUrl(), "unexpected empty request type URL: %q", resp.GetDeltaRequest().GetTypeUrl())
 	out, err := resp.GetDeltaDiscoveryResponse()
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(out.GetResources()) != len(resources) {
-		t.Errorf("unexpected number of responses: got %d, want %d", len(out.GetResources()), len(resources))
-	}
+	require.NoError(t, err)
+	assert.Equalf(t, len(out.GetResources()), len(resources), "unexpected number of responses: got %d, want %d", len(out.GetResources()), len(resources))
```

Comment:
Iirc assert usually expects `(t, expected, actual)`, but in some cases the opposite is set here. Not a big deal at all though